src/client.ts

```diff
@@ -135,14 +136,21 @@ export class GenesysCloudWebrtcSdk extends (EventEmitter as { new(): StrictEvent
 
     // Telemetry for specific events
     // onPendingSession, onSession, onMediaStarted, onSessionTerminated logged in event handlers
-    this.on('sdkError', this.logger.error.bind(this.logger));
+    this.on('sdkError', (error: SdkError) => {
+      /* logging errors is dangerous because they could contain PII */
+      this.logger.error(error, null, true);
+    });
     this.on('disconnected', this.logger.error.bind(this.logger, 'onDisconnected'));
-    this.on('cancelPendingSession', this.logger.info.bind(this.logger, 'cancelPendingSession'));
-    this.on('handledPendingSession', this.logger.info.bind(this.logger, 'handledPendingSession'));
-    this.on('trace', this.logger.info.bind(this.logger));
```

Comment:
I couldn't find anything useful as to what is in these traces. I couldn't even find any reference to `traceRtcSession` (which is what this is proxied from) in `stanza`. I figured if we weren't currently using this, we wouldn't miss it.